src/providers/definition/viewCodeActionProvider.ts

```diff
@@ -5,45 +5,45 @@ import * as definitionProviderHelper from './definitionProviderHelper';
 import { viewSuggestions as suggestions } from './common';
 
 export class ViewCodeActionProvider implements vscode.CodeActionProvider {
-	public async provideCodeActions (document: vscode.TextDocument, range: vscode.Range): Promise<vscode.CodeAction[]> {
+	public async provideCodeActions(document: vscode.TextDocument, range: vscode.Range): Promise<vscode.Command[]> {
```

Comment:
Should this function delegate to the `provideCodeActions` function in `definitionProviderHelper.ts` or should that be removed?